geotrek/api/locale/fr/LC_MESSAGES/django.po

```diff
@@ -638,3 +629,14 @@ msgid ""
 msgstr ""
 "Ce champ est déprécié et sera retiré dans de prochaines releases, merci "
 "d'utiliser 'capacity' à la place."
+
+#, fuzzy
```

Comment:
fuzzy significate that django has tried to translate itmself. Please check and correct translate, then delete fuzzy reference